pennylane/hf/tapering.py

```diff
@@ -217,84 +231,26 @@ def generate_paulis(generators, num_qubits):
     >>> generators = [qml.Hamiltonian([1.0], [qml.PauliZ(0) @ qml.PauliZ(1)]),
     ...               qml.Hamiltonian([1.0], [qml.PauliZ(0) @ qml.PauliZ(2)]),
     ...               qml.Hamiltonian([1.0], [qml.PauliZ(0) @ qml.PauliZ(3)])]
-    >>> generate_paulis(generators, 4)
+    >>> paulix_ops(generators, 4)
     [PauliX(wires=[1]), PauliX(wires=[2]), PauliX(wires=[3])]
     """
     ops_generator = [g.ops[0] if isinstance(g.ops, list) else g.ops for g in generators]
     bmat = _binary_matrix(ops_generator, num_qubits)
 
-    paulix_ops = []
+    paulixops = []
     for row in range(bmat.shape[0]):
         bmatrow = bmat[row]
         bmatrest = np.delete(bmat, row, axis=0)
-        # reversing the order to priortize removing higher index wires first
+        # reversing the order to prioritize removing higher index wires first
         for col in range(bmat.shape[1] // 2)[::-1]:
             # Anti-commutes with the (row) and commutes with all other symmetries.
             if bmatrow[col] and np.array_equal(
                 bmatrest[:, col], np.zeros(bmat.shape[0] - 1, dtype=int)
             ):
-                paulix_ops.append(qml.PauliX(col))
+                paulixops.append(qml.PauliX(col))
                 break
 
-    return paulix_ops
-
-
-def generate_symmetries(qubit_op, num_qubits):
```

Comment:
This wrapper function was accepting the untapered Hamiltonian and making few calls to return the symmetry generators and paulix ops. The user now directly computes generators and paulix ops, so no need to have thus wrapper function.